### PR TITLE
Include .sh files in samples

### PR DESCRIPTION
**Pull request by shamsher31**
To fix https://github.com/istio/istio/issues/27838

### CODE REVIEW

File: pkg/build/archive.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func Archive(manifest model.Manifest) error {
 
 		// Set up samples. We filter down to only some file patterns
 		// TODO - clean this up. We probably include files we don't want and exclude files we do want.
-		includePatterns = []string{"*.yaml", "*.md", "cleanup.sh", "*.txt", "*.pem", "*.conf", "*.tpl", "*.json", "Makefile"}
+		includePatterns = []string{"*.yaml", "*.md", "*.sh", "cleanup.sh", "*.txt", "*.pem", "*.conf", "*.tpl", "*.json", "Makefile"}
```
Comment (brian-avery) [inline pkg/build/archive.go:63]: Does *.sh cover cleanup.sh?

Comment (shamsher31) [reply pkg/build/archive.go:63]: Ahh, yes it includes the following files.
```
$ find . -type f -name "*.sh"
./bookinfo/build_push_update_images.sh
./bookinfo/platform/kube/cleanup.sh
./bookinfo/src/mongodb/script.sh
./bookinfo/src/build-services.sh
./certs/generate-workload.sh
./helloworld/src/build_service.sh
./helloworld/loadgen.sh
./rawvm/demo.sh
./multicluster/gen-eastwest-gateway.sh
```

**Review by brian-avery — APPROVED** [review]

**Comment by rvennam** [comment]
not seeing this change in today's build https://gcsweb.istio.io/gcs/istio-build/dev/1.8-alpha.734140f47c4d4e33e3c4bc6d5c8b69d184fee7e7

**Comment by howardjohn** [comment]
You need to pull in the update into istio/istio

**Comment by rvennam** [comment]
Thanks https://github.com/istio/istio/pull/27891

**Pull request merged**